packages/main/src/Avatar.ts

```diff
@@ -493,6 +500,32 @@ class Avatar extends UI5Element implements ITabbable, IAvatarGroupItem {
 		}
 		this._imageLoadError = true;
 	}
+
+	_getAriaTypeDescription() {
+		switch (this._ariaHasPopup) {
+		case "dialog":
+			return Avatar.i18nBundle.getText(ARIA_HASPOPUP_DIALOG);
+		case "grid":
+			return Avatar.i18nBundle.getText(ARIA_HASPOPUP_GRID);
+		case "listbox":
+			return Avatar.i18nBundle.getText(ARIA_HASPOPUP_LISTBOX);
+		case "menu":
+			return Avatar.i18nBundle.getText(ARIA_HASPOPUP_MENU);
+		case "tree":
+			return Avatar.i18nBundle.getText(ARIA_HASPOPUP_TREE);
+		default:
+			return "";
+		}
+	}
+
+	get accessibilityInfo() {
+		return {
+			role: this._role as AriaRole,
+			type: this._getAriaTypeDescription(),
```

Comment:
An avatar should not be announced as a dialog, grid, or similar. The type should be set according to the component's actual role.